Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Core/Src/Training.c

```diff
@@ -1,6 +1,65 @@
 #include "Training.h"
+#include <stdio.h>
 
-void trainingTask(uint8_t* data)
-{
-
-}
+void trainingTask(uint8_t* data){
+    uint8_t motor1_forward_or_reverse = data[0] & 0b00000001;
```

Comment:
We prefer using camelCase instead of snake_case.

This is the Solar Car team style guide: https://github.com/UCSolarCarTeam/Development-Resources/tree/master/Development-Guide/Style-Guide

Reply:
Nice naming though - easy to understand!